test/routes/tasklist.test.js

```diff
@@ -85,10 +89,34 @@ describe('GET /tasklist/', () => {
         expect(res.text).toMatch(/Digital Prison Services.+Categorisation dashboard/s)
         expect(res.text).toContain('Categorisation task list')
         expect(res.text).toContain('Offending history')
+        expect(res.text).toContain('Further charges')
         expect(res.text).toContain('Not yet checked')
         expect(res.text).toContain('Conditional Release Date')
         expect(res.text).toContain('04/04/2020')
-      }))
+        expect(data.isFemale).toContain(false)
```

Comment:
This is checking the data you set up. Same below with the other test.
Also constant data is unrelated, i.e. if you change value to true, test will still pass. Happy to discuss.

Reply:
done - painful.